posthog/hogql/ast.py

```diff
@@ -768,7 +768,7 @@ class ExprCall(Expr):
 @dataclass(kw_only=True)
 class JoinConstraint(Expr):
     expr: Expr
-    constraint_type: Literal["ON", "USING"]
+    constraint_type: Literal["ON", "USING"] = "ON"
```

Comment:
Hmm, why do we want to set a default here?

Reply:
Oh yeah, this is unintended